cli/src/helper.rs

```diff
@@ -61,9 +106,9 @@ pub async fn generate_mnemonic() -> Result<String, Error> {
     println_log_info!("Mnemonic has been generated.");
```

Comment:
For me it wasnt clear that the write Select was related to the mnemonic. Lets add
`Mnemonic has been generated. Do you want to: `

Reply:
okay, I understand. but we should probably put in a separate line with just a `println!` so it doesn't get logged.